File: BootcampProjeto1/app/src/main/java/com/example/bootcampprojeto1/MainActivity.java
package com.example.bootcampprojeto1;

import android.os.Bundle;
import android.os.PersistableBundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.example.bootcampprojeto1.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding;
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(this.getLayoutInflater());
        setContentView(binding.getRoot());


    }
}

File: BootcampProjeto1/app/src/main/java/com/example/bootcampprojeto1/MainActivity.kt
package com.example.bootcampprojeto1

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.bootcampprojeto1.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.tvHello.text = "lalala"


    }
}